prdoc/pr_3485.prdoc

```diff
@@ -0,0 +1,18 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Broker: sale price runtime api"
+
+doc:
+  - audience: 
+    - Runtime Dev 
+    - Runtime User
```

Comment:
@bkchr Are multiple audiences supported? Based on this I would assume not: https://github.com/paritytech/polkadot-sdk/blob/463ccb8f9028ebf5f7fc53d7e835c21f43172253/prdoc/schema_user.json#L101